src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -99,6 +99,9 @@ public partial struct AssemblyDefinition
         public System.Reflection.Metadata.StringHandle Name { get { throw null; } }
         public System.Reflection.Metadata.BlobHandle PublicKey { get { throw null; } }
         public System.Version Version { get { throw null; } }
+#if !NETSTANDARD11
+        public System.Reflection.AssemblyName GetAssemblyName() { throw null; }
```

Comment:
To clarify, we could implement a light-up but it's non-trivial. I'll file a separate work item for that.